Upgrage Svelte from `v5.14.1` to `v5.14.4` breaks deployment to Cloudlfare Workers

### Describe the bug

I upgraded path version of Svelte and got the following error during deployment to Cloudflare Workers:

```
21:47:50.861 | npm ERR! code EBADENGINE
21:47:50.863 | npm ERR! engine Unsupported engine
21:47:50.863 | npm ERR! engine Not compatible with your version of node/npm: @typescript-eslint/types@8.18.1
21:47:50.863 | npm ERR! notsup Not compatible with your version of node/npm: @typescript-eslint/types@8.18.1
21:47:50.864 | npm ERR! notsup Required: {"node":"^18.18.0 || ^20.9.0 || >=21.1.0"}
21:47:50.864 | npm ERR! notsup Actual:   {"npm":"9.6.7","node":"v18.17.1"}
```

I guess it's because `@typescript-eslint/types` was added in `v5.14.3` through new `esrap` dependency.

```
┬ svelte@5.14.3
└─┬ esrap@1.3.1
  └── @typescript-eslint/types@8.18.1
```

### Reproduction

I've just upgraded Svelte to patch version and nothing else.

### Logs

_No response_

### System Info

```shell
System:
    OS: Linux 5.15 Ubuntu 22.04.4 LTS 22.04.4 LTS (Jammy Jellyfish)
    CPU: (16) x64 AMD Ryzen 7 3700X 8-Core Processor
    Memory: 12.50 GB / 15.58 GB
    Container: Yes
    Shell: 5.8.1 - /usr/bin/zsh
  Binaries:
    Node: 23.3.0 - /run/user/1000/fnm_multishells/9366_1734546699967/bin/node
    npm: 10.9.1 - /run/user/1000/fnm_multishells/9366_1734546699967/bin/npm
    bun: 1.1.17 - ~/.bun/bin/bun
  npmPackages:
    svelte: ^5.14.4 => 5.14.4
```


### Severity

blocking an upgrade

--- a/.changeset/cool-ducks-carry.md
+++ b/.changeset/cool-ducks-carry.md
@@ -1,0 +1,5 @@
+---
+'svelte': patch
+---
+
+fix: bump esrap dependency

--- a/packages/svelte/package.json
+++ b/packages/svelte/package.json
@@ -147,7 +147,7 @@
     "aria-query": "^5.3.1",
     "axobject-query": "^4.1.0",
     "esm-env": "^1.2.1",
-    "esrap": "^1.3.1",
+    "esrap": "^1.3.2",
     "is-reference": "^3.0.3",
     "locate-character": "^3.0.0",
     "magic-string": "^0.30.11",

--- a/packages/svelte/scripts/process-messages/index.js
+++ b/packages/svelte/scripts/process-messages/index.js
@@ -389,7 +389,6 @@ function transform(name, dest) {
 		ast.body.push(clone);
 	}
 
-	// @ts-expect-error
 	const module = esrap.print(ast);
 
 	fs.writeFileSync(

--- a/packages/svelte/src/compiler/phases/3-transform/index.js
+++ b/packages/svelte/src/compiler/phases/3-transform/index.js
@@ -34,7 +34,6 @@ export function transform_component(analysis, source, options) {
 
 	const js_source_name = get_source_name(options.filename, options.outputFilename, 'input.svelte');
 
-	// @ts-expect-error
 	const js = print(program, {
 		// include source content; makes it easier/more robust looking up the source map code
 		// (else esrap does return null for source and sourceMapContent which may trip up tooling)
@@ -95,7 +94,6 @@ export function transform_module(analysis, source, options) {
 	}
 
 	return {
-		// @ts-expect-error
 		js: print(program, {
 			// include source content; makes it easier/more robust looking up the source map code
 			// (else esrap does return null for source and sourceMapContent which may trip up tooling)

